pkg/auto/wordpress/job/meter.go

```diff
@@ -14,7 +14,7 @@ import (
 
 type listMeterReadingFn func(ctx context.Context, in *gen.ListMeterReadingHistoryRequest, opts ...grpc.CallOption) (*gen.ListMeterReadingHistoryResponse, error)
 
-func getRecordsByTime(ctx context.Context, historyFn listMeterReadingFn, meter string, now time.Time, filterTime time.Duration) ([2]*gen.MeterReadingRecord, error) {
+func getRecordsByTime(ctx context.Context, historyFn listMeterReadingFn, meter string, now time.Time, filterTime time.Duration) (*gen.MeterReadingRecord, *gen.MeterReadingRecord, error) {
```

Comment:
Repeated types in return should be named, otherwise you have no idea what they are.